app/models/user.rb

```diff
@@ -67,6 +67,12 @@ def filter(query)
 
   after_update :when_org_changes
 
+  ##
+  # This method uses Devise's built-in handling for inactive users
+  def active_for_authentication?
+    super and self.active?
```

Comment:
can you please change `and` to `&&` because of this Ruby weirdness: 
https://stackoverflow.com/a/1426835/6234472